benchmarks/kernels/benchmark_sgmv_triton.py

```diff
@@ -0,0 +1,127 @@
+import torch
+import triton
+
+from tests.lora.test_sgmv_triton import MAX_TEST_POWER, setup_
+from vllm.model_executor.layers.lora import sgmv_triton as sgmv
+
+
+@triton.testing.perf_report(
```

Comment:
Looks like triton testing needs both `pandas` and `matplotlib` -- could you add those to `requirements-dev.txt` in the benchmarking section?

Reply:
Sure